src/pocketmine/lang/locale/deu.ini

```diff
@@ -324,16 +324,26 @@ pocketmine.plugin.deprecatedEvent=Plugin {%0} hat einen Listener für '{%1}' auf
 pocketmine.plugin.eventError="Event '{%0}' konnte nicht zu '{%1}' weitergegeben werden: {%2} in {%3}"
 
 # -------------------- AdditionalLanguage --------------------
+pocketmine.command.weather.description=Legt das Wetter fest
 commands.weather.usage=/weather <clear|rain|thunder> [Welt]
 commands.weather.worldnotfound=Die Welt '{%0}' gibt es nicht
 commands.weather.clear=Wechsle zu schönem Wetter
 commands.weather.rain=Wechsle zu Regenwetter
 commands.weather.thunder=Wechsle zu Gewitter
 
+pocketmine.command.xp.description=Legt das Erfahrungslevel von Spielern fest
```

Comment:
Mal so ne frage:
ist es nicht so, dass wenn man /xp add player 5 macht, es erfahrung ist, und /xp add player 5L level?
Wenn ja, stimmt in all diesen 4 übersetzungen der inhalt nicht. (ps: warum nicht für paar commands originale bukkit-translation nutzen? sollten kompatibel sein)